packages/fx-core/resource/package.nls.json

```diff
@@ -901,6 +901,7 @@
   "driver.apiKey.error.clientSecretInvalid": "Invalid client secret. It should be 10 to 512 characters long.",
   "driver.apiKey.error.domainInvalid": "Invalid domain. Please follow these rules: 1. Max %d domain(s) per API key. 2. Use comma to separate domains.",
   "driver.apiKey.error.failedToGetDomain": "Unable to get domain from API specification. Make sure your API specification is valid.",
+  "driver.apiKey.error.authMissingInSpec": "No API in the OpenAPI specification file contains API key authentication with the name %s.",
```

Comment:
Proposed content: No API in the OpenAPI specification file matches the API key name '%s'. Please verify the key name.

Reply:
Thanks, updated